cmd/kyma/sync/function/function.go

```diff
@@ -30,20 +30,24 @@ func NewCmd(o *Options) *cobra.Command {
 		Long: `Use this command to download the Function's code and dependencies from the cluster to create or update these resources in your local workspace.
 Use the flags to specify the name of your Function, the Namespace, or the location for your project.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return c.Run()
+			return c.Run(args[0])
+		},
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return errors.New("missing name of the function")
+			}
+			return nil
```

Comment:
```suggestion
		Args: cobra.ExactArgs(1)
```

Reply:
Only suggested if error message is not so relevant, otherwise please ignore.

Reply:
I didn't use `ExactArgs` on purpose to have the custom error msg, but I will change it if you insist on it

Reply:
As I said, only suggested if error message was is not relevant. If it is, leave it as it is.